src/brain_teasers/python/array_product/src/array_product.py

```diff
@@ -0,0 +1,34 @@
+def product_of_array_except_self(arr):
```

Comment:
Your code looks nice and it passed the test, It is only the module name that needs to be added on top, and matching the function name with the Python file name array_product

Reply:
I should have thought about tit in advance. I changed the function name in to array_product and updated readme and unit-tests accordingly.